mitiq/lre/multivariate_scaling/layerwise_folding.py

```diff
@@ -89,7 +89,7 @@ def _get_chunks(
     ]
 
 
-def _get_scale_factor_vectors(
+def get_scale_factor_vectors(
```

Comment:
Reminder to myself that I will need to convert this function to allow for arbitrary `QPROGRAM`.